boards/avsextrem/Makefile.features

```diff
@@ -1,9 +1,7 @@
 # Put defined MCU peripherals here (in alphabetical order)
-FEATURES_PROVIDED += periph_gpio periph_gpio_irq
-FEATURES_PROVIDED += periph_rtc
-FEATURES_PROVIDED += periph_spi
-FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_pwm
```

Comment:
`avsextrem` has no PWM configured in [`periph_conf.h`](https://github.com/RIOT-OS/RIOT/blob/master/boards/avsextrem/include/periph_conf.h)

Reply:
This is super strange. If I recall the email correctly, it is basically an MSB-A2 with an "shield" on top...

Reply:
Dropped the PWM and sorted the entries alphabetically

Reply:
Looks a bit different on the [flyer](http://www.mi.fu-berlin.de/inf/groups/ag-tech/projects/AVS-Extrem/AVS-ExtremFlyer.pdf)